projects/Mallard/src/services/__tests__/errors.spec.ts

```diff
@@ -151,4 +151,11 @@ describe('errorService', () => {
 
         expect(Sentry.captureException).toHaveBeenCalledTimes(2)
     })
+
+    it('should send handled exception to Crashlytics', () => {
+        // const crashlytics = errorService.crashlytics
+        errorService.captureException(new Error())
+        // TODO can't make it work right now
+        expect(errorService.crashlytics.recordError).toBeCalled()
```

Comment:
@jimhunty can't make it work right now, for some reason `crashlytics` is not getting called, must be similar case where we noticed earlier that `crashlytics` instance is different. Would you mind having a look at this or we pair on this?